### PR TITLE
improvements Dantooine Blazor BFF

### PR DESCRIPTION
**Pull request by damienbod**
- remove PWA bits, not used
- Use authorized handler for HttpClients when authorized
- Only use YARP for required path(s)

Some improvements , Greetings Damien

### CODE REVIEW

File: samples/Dantooine/Dantooine.Client/Server/appsettings.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
         "ClusterId": "cluster1",
         "AuthorizationPolicy": "CookieAuthenticationPolicy",
         "Match": {
-          "Path": "{**catch-all}"
+          "Path": "api/DantooineApi1" // "{**catch-all}"
```
Comment (kevinchalet) [inline samples/Dantooine/Dantooine.Client/Server/appsettings.json:18]:
```suggestion
          "Path": "api/DantooineApi1"
```

**Review by kevinchalet — APPROVED** [review]
Thanks for your PR!

File: samples/Dantooine/Dantooine.Client/Server/appsettings.json (hunk 1 above)
Comment (damienbod) [reply samples/Dantooine/Dantooine.Client/Server/appsettings.json]: fixed

**Comment by kevinchalet** [comment]
Merged, thanks! 😃

**Pull request merged**